jormungandr-lib/src/time.rs

```diff
@@ -262,6 +273,18 @@ impl From<u64> for SystemTime {
     }
 }
 
+impl From<u64> for SecondsSinceUnixEpoch {
+    fn from(seconds_since_epoch: u64) -> Self {
+        SecondsSinceUnixEpoch(seconds_since_epoch)
+    }
+}
+
+impl From<SecondsSinceUnixEpoch> for u64 {
+    fn from(seconds_since_epoch: SecondsSinceUnixEpoch) -> Self {
+        seconds_since_epoch.0
+    }
+}
+
 impl From<time::Duration> for Duration {
```

Comment:
And why do we even have our own `Duration`...

Reply:
Probably for Serde support